src/MoviesContainer/MoviesContainer.js

```diff
@@ -1,9 +1,21 @@
 import './MoviesContainer.css';
 
-function Movies() {
+function Movies({posters}) {
```

Comment:
well revisit in the future to determine what qualities we willbe passing in as props.